### DISCUSSION TITLE
Price import from CSV with Datahub

### DISCUSSION THREAD

**Discussion by twin-elements** [coreshop/CoreShop#2596] [Q&A] [answered]
Hi. 
Does Coreshop somehow allow you to import product prices using Datahub import?

**Comment by dpfaffenbauer** [comment, marked as answer]
Hi,

technically it does. Practically we probably need to implement the Data Type somehow. We don't use Datahub Importer, we use Data Definitions were we have integrations (https://github.com/instride-ch/datadefinitions)
